src/connectors/aws_collect.py

```diff
@@ -1685,6 +1729,7 @@ async def aioingest(table_name, options, dryrun=False):
             'iam.list_roles',
             'inspector.list_findings',
             'iam.list_groups',
+            'ebs.describe_snapshot',
```

Comment:
the API being ingested is `ec2 describe-snapshots`, so this should be `ec2.describe_snapshots`